### PR TITLE
Add custom habitat content in listing directory DAH-1050

### PR DESCRIPTION
**Pull request by akegan**
DAH-1050

Review instructions are in the ticket.

### CODE REVIEW

File: app/javascript/pages/ListingDirectory/DirectoryHelpers.tsx [hunk 1]
```diff
@@ -190,45 +191,61 @@ export const getTableSubHeader = (listing: RailsRentalListing) => {
...
-        responsiveCollapse: true,
-        cellClassName: "px-5 py-3",
-        headersHiddenDesktop: ["availability"],
-        stackedData: stackedDataFxn(listing),
-      }}
-      seeDetailsLink={`/listings/${listing.listingID}`}
-    />
-  ))
+  listings.map((listing: Listing, index) => {
+    const hasCustomContent = listing.Reserved_community_type === "Habitat for Humanity"
```
Comment (jacksongonzales) [inline app/javascript/pages/ListingDirectory/DirectoryHelpers.tsx:207]: not sure what the scale of these kinds of inlined strings are, but I've found placing values like this in a `.env` file or the like helpful

Comment (akegan) [reply app/javascript/pages/ListingDirectory/DirectoryHelpers.tsx:207]: yes, good point! I'll at least move it to the top of this file as a const we can re-use elsewhere. We'll also have to put this and others in a map for DAH-1010 which should help

File: app/javascript/pages/ListingDirectory/DirectoryHelpers.tsx [hunk 2]
```diff
@@ -190,45 +191,61 @@ export const getTableSubHeader = (listing: RailsRentalListing) => {
   return null
 }
 
+// Get imageCardProps for a given listing
+const getImageCardProps = (listing, hasFiltersSet?: boolean) => ({
+  imageUrl: listing.imageURL,
+  subtitle: `${listing.Building_Street_Address}, ${listing.Building_City} ${listing.Building_State}, ${listing.Building_Zip_Code}`,
```
Comment (jacksongonzales) [inline app/javascript/pages/ListingDirectory/DirectoryHelpers.tsx:197]: Screen Shot 2022-01-25 at 11 26 27 AM
any ideas for a less technical message we could fallback on in cases like this?

Comment (akegan) [reply app/javascript/pages/ListingDirectory/DirectoryHelpers.tsx:197]: there is also a ticket for that! DAH-1008

Comment (jacksongonzales) [reply app/javascript/pages/ListingDirectory/DirectoryHelpers.tsx:197]: good-first-ticket!!

File: app/javascript/pages/ListingDirectory/DirectoryHelpers.tsx (hunk 1 above)
Comment (akegan) [reply app/javascript/pages/ListingDirectory/DirectoryHelpers.tsx]: just pushed this fix

**Review by jacksongonzales — APPROVED** [review]

**Pull request merged**